### ISSUE TITLE
Implement polls like they have at mastodon

### ISSUE THREAD

**Issue by TheVillageGuy** [MbinOrg/mbin#647]
This example says it all

https://kzoo.to/@jeze/112170632611266384

This is what we get on mbin

https://rimworld.gallery/m/debug/t/787/Elley-Smith-vote-at-jeze-at-kzoo-to

**Comment by BentiGorlich** [comment]
A quick explainer why we do get the post but not the poll: the poll is just an attachment like an image for example, so software that does not support a particular attachment just sees the post without it.

I think this would be a cool addition, however I don't know how I would prioritize it. We have so much to work on, that I feel this is not important enough to consider right now (so I would tend towards a low priority), but I feel like it has to be disussed.

How do you feel @melroy89 @nobodyatroot @e-five256 @asdfzdfj about it?

**Comment by BentiGorlich** [comment]
Of course, why would we do anything else :sweat_smile:

**Comment by melroy89** [comment]
Agreed would be cool. So this will also show-up: https://fedia.io/m/random/p/874855/Previous-Action-Video-of-the-previous-45-frames-https-tomkahe-com-system-media-attachments-files-113-214-228-881-459-942-original-33e41a74ec94603c-mp4#post-comment-1430305
